File: src/Other/Internals/Search/SearchKeywordAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpenSilver.Samples.Showcase.Search
{
    [AttributeUsage(AttributeTargets.Class, AllowMultiple = false)]
    public class SearchKeywordsAttribute : Attribute
    {
        public string[] Keywords { get; }

        public SearchKeywordsAttribute(params string[] keywords)
        {
            Keywords = keywords;
        }
    }
}

File: src/OpenSilver.Showcase/Other/Internals/Search/SearchKeywordAttribute.fs
namespace OpenSilver.Showcase.Search

open System

[<AllowNullLiteral>]
[<AttributeUsage(AttributeTargets.Class, AllowMultiple = false)>]
type SearchKeywordsAttribute([<ParamArray>] keywords: string[]) =
    inherit Attribute()

    member _.Keywords = keywords
